commit c6d91c3ed04d19852d44bf4ee51461f5ac22c752 (refs/changes/77/2477/3)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2017-08-30T19:13:15+00:00 (2 years, 1 month ago)
    
    OS-6317 new LTP failure: keyctl02
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/usr/src/lib/brand/lx/testing/ltp_skiplist b/usr/src/lib/brand/lx/testing/ltp_skiplist
index 571717b23a..48d56f180c 100644
--- a/usr/src/lib/brand/lx/testing/ltp_skiplist
+++ b/usr/src/lib/brand/lx/testing/ltp_skiplist
@@ -172,6 +172,10 @@ ioctl04
 ioctl05
 ioctl06
 keyctl01
+keyctl02
+keyctl03
+keyctl04
+keyctl05
 kcmp01
 kcmp02
 kcmp03
